enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/context.tsx

```diff
@@ -75,7 +76,7 @@ export const InteractiveQuestionProvider = ({
         onReset: onReset || resetQuestion,
         onNavigateBack,
         mode,
-        plugins,
+        plugins: initializePlugins(plugins),
```

Comment:
I assume you initialize it here too because there could be more initializers later on, now we only have dashboard initializer.

I'd put off this change, until we actually have other initializer, since I'm not even sure if this is the way we will end up with for initializing the plugin values.

For example, instead of having to call `initializePlugins` which would theoretically initialize all plugins, but would only be used for a subset of it i.e. we won't care about dashboard plugins here since it's question context, we could add default values for each context directly e.g. `plugins: initializeDashboardPlugins(plugins)`  That way we could initialize different default values based on what context we're using. I find that slightly better since it reduces the scope of the variables we're dealing with (rather than getting the big object `plugins` we get different default values based on what context we're working with).

This is somewhat related to the [plugin scope](https://github.com/metabase/metabase/pull/45138#discussion_r1668067683) I mentioned earlier.